Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -272,9 +278,12 @@ private static GameObject CreateFire(Fire vob, GameObject parent = null)
             if (vob.VobTree == "")
                 return go;
 
-            var vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree);
+            if (!vobTreeCache.TryGetValue(vob.VobTree.ToLower(), out IWorld vobTree))
+            {
+                vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree).Cache();
```

Comment:
```suggestion
                vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree);
```

Reply:
We can keep this cache. At least the .zen itself isn't re-parsed every time.